packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,123 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
 pnpm add @idos-network/idos-issuer-sdk-js
 ```
 
+
+## Quickstart
+
+Create an issuer config with your secret key. This config will be used to interact with the idOS.
+```js
+import { createIssuerConfig } from "@idos-network/idos-issuer-sdk-js";
+import { Wallet } from "ethers";
+
+const wallet = new Wallet("YOUR_PRIVATE_KEY");
+
+const issuerConfig = await createIssuerConfig({
+  nodeUrl: "https://node.url",
+  secretKey: "YOUR_SECRET_KEY",
+  signer: wallet // Or use a KeyPair from near-api-js
+});
+```
+### Creating a human profile.
+
+In order to create a human profile in idOS, you need to have a wallet associated with the human and a public key for the human.
+Passing ids are optional and will be auto-generated if not provided.
+
+```js
+import { createHuman } from "@idos-network/idos-issuer-sdk-js";
+import issuerConfig from "./issuer-config.js";
+
+const human = {
+  id: crypto.randomUUID(),
+  current_public_key: "DERIVED_PUBLIC_KEY",
+}
+
+const walletPayload = {
+  id: crypto.randomUUID(),
+  address: "0x0",
+  wallet_type: "EVM",
+  message: "SIGN_MESSAGE",
+  signature: "DERIVED_SIGNATURE",
+  public_key: "DERIVED_PUBLIC_KEY",   
+}
+
+// Will return a tuple with the human and the wallet associated to the human.
+const [profile, wallet] = await createHuman(issuerConfig, human, walletPayload); 
+```
+
+## Writing credentials.
+
+In order to write a credential to idOS, the issuer needs to obtain permission from the user. This can be done in two ways: either through a permissioned approach or by using a write grant. Below are the two methods for writing credentials.
+
+#### 1. Using Write Grant
+The first method involves getting permission from the user via a Write Grant. This grants the issuer the ability to write to the user’s profile. To do this, you must first create a Write Grant using the idOS SDK.
+
+Example:
+<div >
+ you can create a write grant by calling the  <a href="https://github.com/idos-network/idos-sdk-js/tree/main/packages/idos-sdk-js#write-grants">addWriteGrant</a>
+</div>
```

Comment:
Why the manual `<div>`? Shouldn't an empty line achieve the same result? :thinking: